test/box-luatest/gh_9247_bsize_on_concurrent_dml_and_alter_test.lua

```diff
@@ -0,0 +1,95 @@
+local server = require('luatest.server')
+local t = require('luatest')
+local g = t.group('gh-9247', {{rollback = false}, -- test box.commit()
+                              {rollback = true}}) -- test box.rollback()
```

Comment:
Parameterizing the test by "rollback" flag doesn't look right. You should have different test cases, e.g. successful truncate, rolled back truncate, etc. Rationale: whether to rollback or not rollback is a part of a test case; there may be test cases in a group that don't have this property (can't be rolled back) or have other properties (e.g. fail by error). If you need to reuse the code between test cases, you should use a function, not a test group parameter.

I'm leaving this up to you because (1) this is just a test and (2) abusing test group paremeterization seems to have become a popular misconception. I'm just saying that it doesn't look right to me.

Reply:
Done.